fix(04-24): add the new BDK PR Review Club Meetings

storopoli opened the pull request:
This is the very very last thing.
I don't want to discuss it.
Just an announcement.
Good to have it here since people that do not attend can always check the site.

jaonoctus reviewed (changes requested):
nACK. Could you please post the original link?

storopoli commented:
> nACK. Could you please post the original link instead of the discord one?

There's no original link. The PR Review Club Meeting lives under `Events` in the BDK's Discord channel and the meeting will be held in the General Audio/Video channel in BDK's Discord server.
I've added commits to change the link to the Discord server invitation link and also added a parenthesis saying that it is under the `Events`.
